src/App.js

```diff
@@ -41,7 +42,11 @@ function App() {
     setTodoList(newTodoList);
   };
 
+  
   return (
+    <BrowserRouter>
```

Comment:
fix indentation in return. switch should be indented one space from BrowserRouter, and route one in from that, then the body inside of route should be indented one from that.